processing/src/main/java/io/druid/segment/data/CompressedVSizeIntsIndexedSupplier.java

```diff
@@ -312,6 +324,12 @@ protected int _get(int index)
       // removes the need for padding
       return buffer.get(buffer.position() + index) & 0xFF;
     }
+
+    @Override
+    public void inspectRuntimeShape(RuntimeShapeInspector inspector)
+    {
+      inspector.visit("buffer", buffer);
```

Comment:
Yes, that's bad that buffer couldn't be properly inspected here. Probably it could be improved later.

> Also 'bigEndian' is changed during processing when loadBuffer() is called.
Thanks, removed inspection of `bigEndian` too.